modules/local/medaka.nf

```diff
@@ -0,0 +1,50 @@
+// Import generic module functions
+include { getSoftwareName } from './functions'
+
+process MEDAKA{
+    tag "$sample_name - Segment:$segment - Ref ID:$id"
+    label 'process_low'
+
+    conda (params.enable_conda ? 'bioconda::medaka=1.4.4' : null)
+    if (workflow.containerEngine == 'singularity' && !params.singularity_pull_docker_container) {
+      container 'https://depot.galaxyproject.org/singularity/medaka:1.4.4--py38h130def0_0'
+    } else {
+      container 'quay.io/biocontainers/medaka:1.4.4--py38h130def0_0'
+    }
+
+    input:
+    tuple val(sample_name), val(segment), val(id), path(fasta), path(bam),
+    path(depths)
+
+    output:
+    tuple val(sample_name), val(segment), val(id), path(fasta), path(depths), path(vcf), emit: vcf
+    path (medaka_dir) , emit: output_dir
+    path (medaka_log), emit: log
+    path "versions.yml" , emit: versions
+
+    script:
+    def software  = getSoftwareName(task.process)
+    vcf           = "${sample_name}.Segment_${segment}.${id}.medaka_variant.vcf"
```

Comment:
```nextflow
def prefix = "${sample_name}.Segment_${segment}.${id}"
vcf = "${prefix}.medaka_variant.vcf"
// I think the trailing "/" might be necessary for Nextflow to treat the path as a directory path rather than  a file path
medaka_dir = "${prefix}.medaka_variant/"
```

Reply:
I did not see difference but just for sure I added trailing "/'